Dangerfile

```diff
@@ -93,11 +93,20 @@ has_license_changes = didModify(["LICENSE"])
 @has_googleutilities_changes = hasChangesIn("GoogleUtilities/")
 @has_zipbuilder_changes = hasChangesIn("ZipBuilder/")
 
+sdk_changes = (git.modified_files + git.added_files + git.deleted_files).select do |line|
+  line.end_with?(".h") || line.end_with?(".m") || line.end_with?(".mm")
```

Comment:
.cc also?

Reply:
Good catch, added that to the check.

Reply:
There's a bunch of others:
.swift
.py
.rb
.sh
.cmake
CMakeLists.txt  <-- not an extension, actual filename

We have a list in https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/check_copyright.sh#L25

Reply:
For the sake of checking SDK changelogs, though, are `.py`, `.rb`, `cmake`, and `.sh` files necessary? I had thought about Swift too but iirc our only Swift library is the Firestore extensions library, and that one doesn't have an associated changelog.

Part of the goal of this PR is to not generate changelog warning checks for scripts and podspecs because most of the time those don't warrant changelog entries (unless I'm mistaken).

Reply:
Agreed, changes to files with those extensions are infrastructure that typically won't require a Changelog update.

Reply:
I see. FirebaseFirestoreSwift will soon have a changelog so I think it's reasonable to nag .swift changes.